overlord/devicestate/devicemgr.go

```diff
@@ -60,6 +61,10 @@ type DeviceManager struct {
 	becomeOperationalBackoff     time.Duration
 	registered                   bool
 	reg                          chan struct{}
+
+	// XXX: is that the best name?
+	// runMode of a UC20 system. "","run","install","recover"
+	runMode string
```

Comment:
this should probably go at the top of the struct.  About the name runMode is clashing with "run mode" itself, I would call it just mode, or operatingMode

Reply:
Thanks, I really like operatingMode